src/main/java/org/cactoos/collection/CollectionEnvelope.java

```diff
@@ -41,6 +43,14 @@
      */
     private final Collection<X> col;
 
+    /**
+     * Ctor.
+     * @param scalar Scalar to return collection
+     */
+    public CollectionEnvelope(final Scalar<Collection<X>> scalar) {
```

Comment:
@fanifieiev please don't do that, it makes no sense for the envelope to take a scala

Reply:
@victornoel Then several classes under `org.cactoos.collection` will suffer.

Reply:
@victornoel I will fix that the other way, ignore above message

Reply:
@victornoel  @paulodamaso I would also like to raise a discussion regarding which collections(lists, sets, etc) should be immutable and which not.
What I can see now is just a mess, some of them are read-only and some not.
What do you think ???

Reply:
@fanifieiev please see the discussions in #1242 for answers to your questions